src/components/AccountsPage/AccountRowItem/index.js

```diff
@@ -104,14 +104,25 @@ const mapDispatchToProps = {
   openModal,
 }
 
+const expandedStates: { [key: string]: boolean } = {}
+
 class AccountRowItem extends PureComponent<Props, State> {
-  state = {
-    expanded: false,
+  constructor(props) {
+    super(props)
+    const { account, parentAccount } = this.props
+    const accountId = parentAccount ? parentAccount.id : account.id
+
+    this.state = {
+      expanded: expandedStates[accountId],
+    }
   }
 
   componentWillReceiveProps(nextProps: Props) {
+    const { account, parentAccount } = nextProps
+    const accountId = parentAccount ? parentAccount.id : account.id
+
     this.setState({
-      expanded: !!nextProps.search,
+      expanded: expandedStates[accountId] || !!nextProps.search,
```

Comment:
so same here, shouldn't we just don't do anything in case account is not an expandable?